stockservice-go/internal/core/ports/ports.go

```diff
@@ -0,0 +1,47 @@
+package ports
```

Comment:
so basically this class serves as an instancier for all the communication points between all the components of the web socket?